pipeline/utils.py

```diff
@@ -0,0 +1,264 @@
+import os
+from typing import Optional
+from configparser import ConfigParser
+from datetime import date
+
+from azure.identity import DefaultAzureCredential, ClientSecretCredential
+from azure.storage.blob import BlobServiceClient
+from pyspark.sql import SparkSession, DataFrame
+
+
+def is_running_in_databricks() -> bool:
+    """Detect whether the current enviroment is in databricks"""
+    return "DATABRICKS_RUNTIME_VERSION" in os.environ
+
+
+def load_config():
+    config = ConfigParser()
+    config.read("config.ini")
+    return config
+
+
+def get_default_blob_account_name() -> str:
+    config = load_config()
+    try:
```

Comment:
Instead of putting the code in try block, try using:
config = load_config()
account_name = config.get("blob", "account_name", fallback="developmentstorageccs")
return account_name

Reply:
Thank you. I changed all similar case to the fallback pattern.